tree/ntupleutil/v7/src/RNTupleImporter.cxx

```diff
@@ -110,11 +153,12 @@ ROOT::Experimental::RResult<std::unique_ptr<ROOT::Experimental::RNTupleImporter>
 ROOT::Experimental::RNTupleImporter::Create(TTree *sourceTree, std::string_view destFileName)
 {
    auto importer = std::unique_ptr<RNTupleImporter>(new RNTupleImporter());
-   importer->fNTupleName = sourceTree->GetName();
-   importer->fSourceTree = sourceTree;
+   auto treeName = sourceTree->GetName();
+   importer->fNTupleNames[treeName] = treeName;
+   importer->fSourceTrees.emplace_back(sourceTree);
 
    // If we have IMT enabled, its best use is for parallel page compression
-   importer->fSourceTree->SetImplicitMT(false);
+   importer->fSourceTrees.at(0)->SetImplicitMT(false);
```

Comment:
Why is this not:
```suggestion
   sourceTree->SetImplicitMT(false);
```
It sounds like the existing wouldn't have the desired effect except for the first tree.